types/model.d.ts

```diff
@@ -19,17 +19,17 @@ export type PropertyType =
  * Property definition
  */
 export interface PropertyDefinition extends AnyObject {
-  name: string;
   type?: PropertyType;
+  id?: boolean;
```

Comment:
In LB3, `id` is a `boolean` or `number`. See https://github.com/strongloop/loopback-datasource-juggler/blob/master/lib/model-definition.js#L132

Reply:
Good catch! Fixed.